compiler/rustc_infer/src/infer/error_reporting/need_type_info.rs

```diff
@@ -1,12 +1,13 @@
 use crate::infer::type_variable::TypeVariableOriginKind;
 use crate::infer::InferCtxt;
+use hir::LocalSource;
```

Comment:
move this back into the `use rustc_hir::{` below

Reply:
This is restored.